### PR TITLE
Astro June 23rd release notes

### PR DESCRIPTION
**Pull request by jwitz**
Based on https://app.zenhub.com/workspaces/developer-productivity-and-readiness-6242176c89ed9000197ec734/reports/release?release=Z2lkOi8vcmFwdG9yL1JlbGVhc2UvNzg2MDU

### CODE REVIEW

File: astro/release-notes.md [hunk 1]
```diff
@@ -16,6 +16,23 @@ If you have any questions or a bug to report, don't hesitate to reach out to [As
...
+### Support for more GCP regions
+
+You can now [create an Astro cluster on GCP](create-cluster.md) in the following regions:
+
+- `asia-southeast1` (Singapore)
+- `australia-southeast1` (Sydney)
+- `europe-west1` (Belgium)
+- `europe-west2` (England)
+- `europe-west3` (Frankfurt)
+- `southamerica-east1` (Sau Paolo)
```
[the remark this answers is not in the document]
Comment (chomipi88) [reply astro/release-notes.md:30]: @dnarain-astro have these been tested and ready to advertise?

**Review by chomipi88 — COMMENTED** [review]
Comment left for Deepak. Otherwise LGTM

**Comment by dnarain-astro** [comment]
We are good to go on these, QA has tested them.

File: astro/release-notes.md [hunk 2]
```diff
@@ -16,6 +16,23 @@ If you have any questions or a bug to report, don't hesitate to reach out to [As
 
 **Latest CLI Version**: 1.1.0 ([Release notes](cli/release-notes.md))
 
+## June 23, 2022
+
+### Support for more GCP regions
```
Comment (paolaperaza) [inline astro/release-notes.md:21]:
```suggestion
### New GCP regions available
```

Consistent with: https://docs.astronomer.io/astro/release-notes#new-aws-regions-available

**Review by paolaperaza — APPROVED** [review]

**Pull request merged**